test/cpp/jit/test_misc.h

```diff
@@ -909,15 +909,101 @@ static void checkShape(
   ASSERT_EQ(ptp->sizes().concrete_sizes().value(), expected);
 }
 
-std::vector<std::size_t> values_to_value_ids(
+std::vector<std::string> values_to_value_ids(
```

Comment:
`parseIR` populates a value map if you give it the appropriate argument. That's probably a more robust way of getting at what you're trying to do here.